app/models/ability.rb

```diff
@@ -78,7 +78,7 @@ def define_school_teacher_abilities(user:, school:)
       school_teacher_can_manage_project?(user:, school:, project:)
     end
     can(%i[read], Project, school_id: school.id, lesson: { visibility: %w[teachers students] })
-    can(%i[read], Project,
+    can(%i[read toggle_finished], Project,
```

Comment:
🔧 its actually students that need to toggle this on any of their projects (i.e. with a `school_id` and `remixed_from_id`)

Reply:
this will need cascading through the tests too 👍